react/orders101/src/features/navbar/Navbar.jsx

```diff
@@ -0,0 +1,42 @@
+import BasketOpen from "../BasketOpen";
+import Hamburger from "../Hamburger";
+
+function Navbar() {
+  return (
+    <div class="nav container">
+      <div class="hamburger__btn-open">
+        <Hamburger />
+      </div>
+      <a class="nav__icons" href="#stores">
+        <div class="nav__icons__item container">
+          <img
+            class="searchicons"
+            src="../src/assets/geo-alt.svg"
+            alt="geo locator"
+          ></img>
+          <span>Stores</span>
+        </div>
+      </a>
+      <div class="logo">
+        <p>DIYToolWorld</p>
+      </div>
+      <a class="nav__icons" href="#ideas">
+        <div class="nav__icons__item container">
+          <img
+            class="searchicons"
+            src="../src/assets/lightbulb.svg"
+            alt="lightbulb"
+          ></img>
+          <span>Ideas & Advice</span>
+        </div>
+      </a>
+
+      <BasketOpen />
+      <div class="nav__search-container">
+        <input class="search-bar" type="search" placeholder="Search..." />
+      </div>
+    </div>
```

Comment:
And after this closing `div` place the Cart instead of keeping it in the `App` component. 

```
<Cart isOpen={isCartOpen} closeCart={closeCart} />
```

(and then wrap everything in a `React.Fragment` I guess)

Reply:
Cart now added in the Nav component and isOpen and closeCart props passed in